src/plugins/intel_npu/src/backend/include/zero_executor.hpp

```diff
@@ -55,7 +59,7 @@ class ZeroExecutor final : public IExecutor {
     }
 
 private:
-    const Config _config;
+    Config _config;
```

Comment:
Btw is there any point in storing the config inside the executor? THere's no getter for it and I've failed to identify any method besides the constructor that uses it. Lemme know if I missed something.